README.md

```diff
@@ -26,7 +28,7 @@ import modin.pandas as pd
 
 #### From PyPI
 
-Modin can be installed with `pip`:
+Modin can be installed with `pip` on Linux, Windows and MacOS:
```

Comment:
We should correct the links to supported engines below. Either to our docs or external ones.
```
Installing from conda forge using `modin-all` will install Modin and 3 engines: ([Ray](https://github.com/ray-project/ray),
[Dask](https://github.com/dask/dask) and [Omnisci](https://modin.readthedocs.io/en/latest/UsingOmnisci/index.html))
```

Reply:
@YarShev Not sure I understand this comment -- I believe the links are already a part of the README in the conda-forge section but you highlighted the PyPI section here?

Reply:
Yes, the links are already a part of the README in the conda-forge section, I just was not able to leave a comment there. I mean we should replace the link to our OmniSci usage section to this one https://www.omnisci.com/platform/omniscidb because the links to Ray and Dask lead to their own docs. Does that make sense?